app/javascript/miq-redux/middleware.ts

```diff
@@ -1,10 +1,11 @@
 import thunk from 'redux-thunk';
 import { routerMiddleware } from 'connected-react-router';
+import { compose } from 'redux';
 
 /**
  * @param history history for redux routing for react components * 
  * @description
  *  routerMiddleware adds middleware which is listening for location changes and updates router state
  *  thunk is a middleware for async redux functions
  */
-export const createMiddlewares = ({ history, middlewares = [] }) => ([routerMiddleware(history), thunk, ...middlewares]);
+export const createMiddlewares = ({ history, middlewares = [] }) => compose(routerMiddleware(history), thunk, ...middlewares);
```

Comment:
Use redux devtools extension here so we can fully use it's power
```javascript
const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
export const createMiddlewares = ({ history, middlewares = [] }) => composeEnhancers(routerMiddleware(history), thunk, ...middlewares);
```

Reply:
Added, thanks! :)